fire/fire_test.py

```diff
@@ -43,6 +43,19 @@ def testFire(self):
     self.assertEqual(fire.Fire(tc.OldStyleWithDefaults, 'double 2'), 4)
     self.assertEqual(fire.Fire(tc.OldStyleWithDefaults, 'triple 4'), 12)
 
+  def testFireDefaultBaseName(self):
+    with mock.patch.object(sys, 'argv',
+                           ['/github/python-fire/fire/base_filename.py']):
```

Comment:
Let's use something like `os.path.join('python-fire', 'fire', 'base_filename.py')` so that we don't assume unix vs windows (/ vs \\). [Also dropping 'github' from the path.] It's not important if it's an absolute or relative path, either's fine.